doc/user-guide/user_guide.md

```diff
@@ -0,0 +1,71 @@
+# DynamoDB Virtual Schema User Guide
+This user guide helps with getting started with the DynamoDB Virtual Schemas.
+
+
+### Installation
+ 
+Upload the latest available release of this adapter to BucketFS.
```

Comment:
Here you can also add a link to Github releases: https://github.com/exasol/dynamodb-virtual-schema/releases

Reply:
Also link to the BucketFS documentation, and short example how to upload jar file to bucket using curl.

> For some users this may be their first encounter with Exasol and may have no idea what is BucketFs or bucket.